changelog/18.0/18.0.0/summary.md

```diff
@@ -101,6 +101,14 @@ Vitess upgrade process from an earlier version if you need to use such a workflo
 Any MoveTables or Migrate workflow that moves a sequence table should only be run after all vitess components have been
 upgraded, and no upgrade should be done while such a workflow is in progress.
 
+#### <a id="vttablet-tx-throttler-dry-run"/>New Dry-run/monitoring-only mode for the transaction throttler
+
+A new CLI flag `--tx-throttler-dry-run` to set the Transaction Throttler to monitoring-only/dry-run mode has been added.
+If the transaction throttler is enabled with `--enable-tx-throttler` and the new dry-run flag is also specified, the
+tablet will not actually throttle any transactions; however, it will increase the counters for transactions throttled
+(`vttablet_transaction_throttler_throttled`). This allows to deploy the transaction throttler in production and gain
```

Comment:
```suggestion
(`vttablet_transaction_throttler_throttled`). This allows us to deploy the transaction throttler in production and gain
```

Reply:
Or `users`. But that verb definitely needs an object.

Reply:
Fixed.